ci/do.sh

```diff
@@ -62,20 +62,47 @@ if [ ! -r ${SPEC_FILE} -o ! -r ${REQ_FILE} ]; then
 fi
 
 if [ "x${IS_CI_BUILD}" != "x1" ] || [ "x${FIRST_TIME_BUILD}" == "x1" -a "x${IS_CI_BUILD}" == "x1" ]; then
-  if [ "x${ID}" == "xcentos" ]; then
+  if [ "x${ID}" == "xcentos" -a "x${VERSION_ID}" == "x7" ]; then
```

Comment:
How about removing `"x${ID}" == "xcentos"` and put a check which works for RHEL and CentOS? as of now, this will work only for centos.

Same for suse check also?